src/OrbitGl/App.h

```diff
@@ -391,7 +391,7 @@ class OrbitApp final : public DataViewFactory, public orbit_capture_client::Capt
     return data_manager_->memory_warning_threshold_kb();
   }
 
-  // TODO(kuebler): Move them to a separate controler at some point
+  // TODO(kuebler): Move them to a separate controller at some point
```

Comment:
Should be a bug link here?

Reply:
A bug id is preferred over ldap, but there are ~20 other locations where an ldap is used. 

I only accidentally saw this typo and decided to fix it. @florian-kuebler if you want to file a bug feel free.

Reply:
Ah, it looked like this is the only place, but I guess there are TODOs that are already formatted correctly :) In that case, keep it as-is, and let's fix usernames later.